setup.cfg

```diff
@@ -1,5 +1,21 @@
 [flake8]
-exclude = build,.git,build-pypy,.tox
+exclude =
+    .git,
```

Comment:
flake8 doesn't complain about any of these things, why do they need to be added to the exclude list?

Reply:
Eliminating unnecessary inspections can reduce the time and computing resources spent in inspections.

Reply:
These are common places that do not need to be checked, and this project is also applicable